spec/image_spec.rb

```diff
@@ -626,5 +626,18 @@ def has_jpeg?
         end
     end
 
+    it 'can has_alpha?' do
+        x = Vips::Image.new_from_file './spec/samples/alpha.png'
+        expect(x.has_alpha?).to be true
+    end
+
+    if Vips::at_least_libvips?(8, 6)
```

Comment:
This is actually never tested on Travis, since there, only libvips 8.5 is being installed. A matrix testing various versions of libvips would be helpful I guess.

Reply:
Good idea, I'll have a look.

Reply:
I was looking up a feature that isn't widely used (maybe because it's little known?), it being build stages. It seem it could also speed up testing a lot: https://docs.travis-ci.com/user/build-stages/warm-cache/

But for the simple case of testing with different libvips versions, I think a simple build matrix where the MAJOR, MINOR, PATCH env variables are being overwritten should suffice.

But testing will take a long time then, since you'd have to build 8.2, 8.3, 8.4, 8.5, 8.6 and soon 8.7 for every Ruby version. This would then be the point where caching would make a lot of sense.